tests/integration/test_experiments.py

```diff
@@ -1,28 +1,32 @@
+from tests.helper import create_random_name
+
+
 def test_experiments(
     finetuner_mocker,
-    first_exp_name='first experiment',
-    second_exp_name='second experiment',
 ):
-    # delete experiments in case there are any
-    finetuner_mocker.delete_experiments()
+    # Always has one default Experiment.
+    assert len(finetuner_mocker.list_experiments()) == 1
+    first_exp_name, second_exp_name = [create_random_name() for _ in range(2)]
     # create an experiment and retrieve it
     finetuner_mocker.create_experiment(name=first_exp_name)
     exp1 = finetuner_mocker.get_experiment(name=first_exp_name)
     assert exp1.name == first_exp_name
     assert exp1.status == 'ACTIVE'
     # create another experiment and list all experiments
     finetuner_mocker.create_experiment(second_exp_name)
-    exps = finetuner_mocker.list_experiments()
-    assert len(exps) == 2
-    experiment_names = sorted([exp.name for exp in exps])
-    assert experiment_names == [first_exp_name, second_exp_name]
-    assert exps[0].status == exps[1].status == 'ACTIVE'
+    experiments = finetuner_mocker.list_experiments()
+    assert len(experiments) == 3
```

Comment:
If we check lengths, we still leave some space for failure. Another CI run might create its own experiments and overall it might be more than 3. Although I'm not sure how frequent this will be so we can leave it this way and observe.